zadani/soft/style.css

```diff
@@ -15,29 +16,311 @@ body {
+    padding-bottom: 20px;
+}
 
+h2 {
+    padding:20px 0px;
+    margin: 0px; 
+    position: relative;
+}
 
+h2::after {
+    content: "";
+    width: 70px;
+    border-bottom: 4px solid rgb(95, 147, 190);
+    position:absolute;
+    bottom:0;
+    left: -30px;
+}
 
+.fotka_islandu  img{
+    width:100%;
+    margin: top 1px;
+    background-size: cover;
+}
+
+.strana_vlevo,
+.strana_vpravo {
+    padding: 15px 25px 15px 25px;
+    font-size: 15px;
+}
+
+.popisek,
+.popisek a{
+    color:rgb(165, 162, 162);
+    font-size: 10px;
+    text-align: center;
+    padding-top:5px;
+    margin:0px;
+}
 
 /* FOTOGALERIE */
 
 
+.galerie {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.foto img  {
+    width:100%;
+    margin-top:  15px;
+    margin-bottom: 10px;
+    display:block;
+}
+
+.popisek_galerie {
+    font-size: 12px;
+    font-family: 'Raleway', serif;
+    font-weight: 700;
+    color:grey;
+    text-align: center;
+}
+.popisek-copy, 
+.popisek-copy a {
+    font-weight: 400;
+    color: rgb(165, 162, 162);
+    font-size: 10px;
+}
+
+.popisek-copy a {
+    text-decoration: underline;
+}
 
 
 /* PATICKA */
+.paticka,
+.paticka a {
+    color: gray;
+    font-size: 10px;
+    padding-bottom: 20px; 
+}
+
+.paticka
+{
+    background-color: #f5f5f54b;
+    font-size: 10px;
+    font-weight: 400;
+    padding: 20px;
+}
+
+
+/* tablet */
+@media only screen and (min-width:500px){
+    .autor::after {
+        width:10%;
+        left: 45%;  
+    }
+
+    .fotka_islandu
+    {
+        position: relative;
+    }
+
+    .menu_seznam li{
+        margin: 30px;
+    }
+
+    .menu
+    {
+        padding: 20px;
+    }
+
+    .popisek {
+        position: absolute;
+        bottom: 15px;
+        right:20px;
+        color:white;
+        font-weight: normal;
+    }
+
+    .galerie {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+
+    .foto {
+        position: relative;
+        overflow: hidden;
+        width: 50%;
+        min-height: 120%;
+    }
+ 
+    .popisek_galerie{
+        width:100%;
+        position:absolute;
+        color:white;
+        background-color: rgba(43, 41, 41, 0.808);
+        transition: 0.2s ease-in-out; 
+    }
+
+    .foto:hover .popisek_galerie {
+        transform: translate(0, -125%);
+        transition: 0.8s;
+    }
+
+    .popisek_galerie a,
+    .popisek a,
+    .popisek-copy {
+        color:white;
+    }
+}
+
+/* PC */
+@media only screen and (min-width:1000px){
+    .hlavicka {
+        width: 100%;
+        height: 500px;
+        padding: 40px 200px 100px 200px;
+    }
+    .cela_stranka{
+        background-color: rgba(224, 215, 215, 0.342);
+        box-shadow: 5px 12px 20px rgba(26, 22, 26, 0.2);
+    }
+
+    .obsah{
+        width: 960px;
+        margin: -50px auto 0 auto;
+        display: flex;
+        flex-direction: column;
+        background-color: white;
```

Comment:
Tohle nemusíme v media query přepisovat, už je to definované v hlavním, mobile-first stylu